draft-ietf-rats-architecture.md

```diff
@@ -249,13 +179,7 @@ The system components perform a series of measurements that may be
 signed by the root of trust, considered as Evidence about the hardware,
 firmware, BIOS, software, etc. that is running.
 
-Attester:
-
-: A device desiring access to a network
-
-Relying Party:
-
-: A network infrastructure device such as a router, switch, or access point
+In this case, the device desiring access to a network must produce attestation for consideration by a network infrastructure device such as a router, switch, or access point
```

Comment:
Not sure I like "produce attestation" as a phrase.  First, it would require "attestation" to be defined in a way that is different from any other use in the document.  Second, it also implies that the device (Attester) actually produce something that is consumed by a router (Relying Party), which is not in the case in this architecture... the Attester produces Evidence for consideration by a Verifier, which produces Attestation Results for consideration by Relying Parties.